cognite/client/utils/_auxiliary.py

```diff
@@ -28,6 +29,10 @@
 THashable = TypeVar("THashable", bound=Hashable)
 
 
+def is_unlimited(limit: Any) -> bool:
```

Comment:
```suggestion
def is_unlimited(limit: Optional[Union[float, int]]) -> bool:
```

Reply:
Btw, what do you think about PEP-484 regarding "Union[float, int]"?

> Rather than requiring that users write import numbers and then use numbers.Float etc., this PEP proposes a straightforward shortcut that is almost as effective: when an argument is annotated as having type float, an argument of type int is acceptable

Reply:
Yeah Optional[float] is sufficient